modules/setting/database.go

```diff
@@ -92,7 +92,7 @@ func loadDBSetting(rootCfg ConfigProvider) {
 	Database.MaxOpenConns = sec.Key("MAX_OPEN_CONNS").MustInt(0)
 
 	Database.IterateBufferSize = sec.Key("ITERATE_BUFFER_SIZE").MustInt(50)
-	Database.LogSQL = sec.Key("LOG_SQL").MustBool(true)
+	Database.LogSQL = sec.Key("LOG_SQL").MustBool(false)
```

Comment:
So this is a break change?

Reply:
I do not think it is "breaking".

The "install.go" already disabled it by default:

```
cfg.Section("database").Key("LOG_SQL").SetValue("false") // LOG_SQL is rarely helpful
```